.github/workflows/goreleaser.yml

```diff
@@ -59,10 +59,8 @@ jobs:
       run: |
         chartVersion=$(echo ${{ github.ref_name }} | cut -c 2-)
 
-        make core-chart IMAGE_TAG=${{chartVersion}}
-
-        sed -i "s/^KUBESTELLAR_VERSION:.*$/KUBESTELLAR_VERSION: \"${chartVersion}\"/g" ${{ env.CORE_CHART_PATH }}/values.yaml
-        sed -i "s/^TRANSPORT_VERSION:.*$/TRANSPORT_VERSION: \"${chartVersion}\"/g" ${{ env.CORE_CHART_PATH }}/values.yaml
+        sed -i "s/^.*KUBESTELLAR_VERSION:.*$/KUBESTELLAR_VERSION: \"${chartVersion}\"/g" ${{ env.CORE_CHART_PATH }}/values.yaml
```

Comment:
Do you really want to remove the anchoring to the start of the line here? And strip off whatever precedes "KUBESTELLAR_VERSION" on the line? Right now there is only one match for this pattern, but this new search&replace seems like it could accidentally make an unwanted change in the future.